### ISSUE TITLE
Generated Links Point to Invalid Target

### ISSUE THREAD

**Issue by Kenji776** [cesarParra/apexdocs#30]
Hey there, this may be a user issue but I don't really see a way to modify this behavior. When I generate markdown files the links that get created point to an invalid path. It seems to be trying to use the root path instead of a relative path. Example when viewing a file located at 

file:///C:/ApexDoc/markdown/Forfeitures/BatchForfeitureEmailNotification.md

the link for the test class mentioned in the @see param pointed at

file:///C:/Forfeitures/BatchForfeitureEmailNotificationTest.md

When it really just needed to point to

../Forfeitures/BatchForfeitureEmailNotificationTest.md

I am using a Chrome extension for my markdown viewer so it's possible that is causing a problem, but then also using another utility to convert the generated markdown files to PDFs the link is also bad there so I think its just being generated using that incorrect path. Again this is probably on me somehow but I'm not quite sure how to fix. Thanks in advance!

**Comment by cesarParra** [comment]
Thanks @Kenji776 . I'll investigate.

**Comment by cesarParra** [comment]
I wasn't able to reproduce the issue with the root path being used, but I did see issues with the path resolution when running the command using the default config (`--group true` and `--targetGenerator jekyll`), while not grouping and/or using `docsify` as the generator seem to work.

I just released version 1.13.7 that hopefully fixes the issues. Things seem to now be working correctly on the [demo site](https://cesarparra.github.io/apexdocs/) with that config and the latest version.
